### PR TITLE
Use correct properties for csv lines to skip

### LINKED ISSUE #10557: Transformation can fail when csv files has a header
**Alluxio Version:**
2.1.0

**Describe the bug**
When a table stores its file as a csv file and it has headers, its property `skip.header.line.count`  is not respected by the transformation and the transformation fails because of type mismatch. 

**To Reproduce**
attach a database with a table that has csv as its data and the csv has a header. 

**Expected behavior**
Expect the transformation to respect the header and the `skip.header.line.count` property

**Urgency**
fix before the next release

**Additional context**

### PR DESCRIPTION
**Pull request by yuzhu**
Previously we were looking for lines to skip in the wrong properties. (serde properties) now look for it in table properties. 

Fixes #10557

### CODE REVIEW

**Comment by yuzhu** [comment]
@chcn @gpang PTAL, thanks

File: table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java [hunk 1]
```diff
@@ -219,6 +219,7 @@ public UdbTable getTable(String tableName) throws IOException {
           .setTableName(tableName)
           .addAllDataCols(HiveUtils.toProto(table.getSd().getCols()))
           .setStorage(HiveUtils.toProto(table.getSd(), pathTranslator))
+          .putAllParameters(table.getParameters())
```
Comment (gpang) [inline table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java]: This was missing from before? This is not related to the serde confusion, right? Is there any unittest that can cover this type of issue?

Comment (yuzhu) [reply table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java]: this is probably best tested using integration test / end to end test. Mocking a hive metaserver to test this has always had the problem of not being very accurate..

**Comment by gpang** [comment]
@yuzhu if this is a bug, can you create an issue, and then link it in the PR description?

**Comment by yuzhu** [comment]
@gpang issue created and referenced

**Comment by yuzhu** [comment]
alluxio-bot, merge this please.

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please.

**Pull request merged**